Sources/Paywall/Paywall/Events/InternalEvents.swift

```diff
@@ -274,23 +274,28 @@ extension Paywall {
         params: params,
         customParams: customParams
       )
-    case .triggerFire(let triggerInfo):
+    case let .triggerFire(triggerInfo, eventData):
       var params: [String: Any] = [:]
       switch triggerInfo {
       case .noRuleMatch:
-        params = ["result": "no_rule_match"]
+        params = [
+          "result": "no_rule_match",
+          "triggerName": eventData.name
```

Comment:
Everything else is snake case, probably worth following the convention

Reply:
Friendly heads up not everything is snakecase, but we definitely should follow a single convention.  

<img width="315" alt="Screen Shot 2022-04-21 at 7 10 19 PM" src="https://user-images.githubusercontent.com/3682820/164582608-3eef580a-d3be-4734-b178-3357c911a9b8.png">

Reply:
I'll create a ticket for @yusuftor .

Reply:
https://linear.app/superwall/issue/SW-863/standardize-all-internalevent-properties-to-snake-case